[2022-10-29 14:30:07,478] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag.also_run_this scheduled__2022-10-28T14:30:00+00:00 [queued]>
[2022-10-29 14:30:07,487] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag.also_run_this scheduled__2022-10-28T14:30:00+00:00 [queued]>
[2022-10-29 14:30:07,487] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-29 14:30:07,488] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-10-29 14:30:07,488] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-29 14:30:07,501] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): also_run_this> on 2022-10-28 14:30:00+00:00
[2022-10-29 14:30:07,503] {standard_task_runner.py:52} INFO - Started process 15547 to run task
[2022-10-29 14:30:07,507] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dag', 'also_run_this', 'scheduled__2022-10-28T14:30:00+00:00', '--job-id', '173', '--raw', '--subdir', 'DAGS_FOLDER/extract_and_load.py', '--cfg-path', '/tmp/tmpr1yr1hlw', '--error-file', '/tmp/tmp885aqa9e']
[2022-10-29 14:30:07,508] {standard_task_runner.py:80} INFO - Job 173: Subtask also_run_this
[2022-10-29 14:30:07,568] {task_command.py:369} INFO - Running <TaskInstance: dag.also_run_this scheduled__2022-10-28T14:30:00+00:00 [running]> on host cf01c04ee00d
[2022-10-29 14:30:07,644] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dag
AIRFLOW_CTX_TASK_ID=also_run_this
AIRFLOW_CTX_EXECUTION_DATE=2022-10-28T14:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-28T14:30:00+00:00
[2022-10-29 14:30:07,645] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-10-29 14:30:07,645] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls']
[2022-10-29 14:30:07,650] {subprocess.py:85} INFO - Output:
[2022-10-29 14:30:07,652] {subprocess.py:96} INFO - Command exited with return code 0
[2022-10-29 14:30:07,685] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dag, task_id=also_run_this, execution_date=20221028T143000, start_date=20221029T143007, end_date=20221029T143007
[2022-10-29 14:30:07,720] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-29 14:30:07,756] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
